packages/apps/human-app/frontend/src/modules/worker/jobs-discovery/helpers/__tests__/should-navigate-to-registration.test.ts

```diff
@@ -0,0 +1,95 @@
+import { describe, expect, it } from 'vitest';
+import { shouldNavigateToRegistration } from '../should-navigate-to-registration';
+import { type Oracle } from '../../../hooks';
+
+describe('shouldNavigateToRegistration Helper', () => {
```

Comment:
Please use faker to generate data.
Also you can DRY oracle data generation to some function in this test file